app/controllers/concerns/curation_concerns/download_behavior.rb

```diff
@@ -40,6 +40,12 @@ def derivative_download_options
       # that files are in a LDP basic container, and thus, included in the asset's uri.
       def authorize_download!
         authorize! :read, params[asset_param_key]
+      rescue CanCan::AccessDenied
+        redirect_to default_image
+      end
+
+      def default_image
+        ActionController::Base.helpers.image_path 'default.png'
```

Comment:
@jcoyne I initially tried that but got `*** NoMethodError Exception: undefined method 'image_path' for #<DownloadsController:0x0000000a898ac8>`